.bids_conversion/convert.sh

```diff
@@ -0,0 +1,192 @@
+#!/bin/bash
+
+set -e -u
+
+# get list of dirs once
+for subdir in [0-9]*; do
+
+    newsub="sub-${subdir}"
+    mkdir -p ${newsub}/ses-3T/anat \
+             ${newsub}/ses-3T/func \
+             ${newsub}/ses-7T/func
+
+    # rename anat files
+    git mv -fk ${subdir}/T1w/T1w_acpc_dc.nii.gz ${newsub}/ses-3T/anat/${newsub}_ses-3T_desc-dc_T1w.nii.gz || true
+    git mv -fk ${subdir}/T1w/T1w_acpc_dc_restore.nii.gz ${newsub}/ses-3T/anat/${newsub}_ses-3T_desc-dcrestore_T1w.nii.gz || true
+    git mv -fk ${subdir}/T1w/T1wDividedByT2w.nii.gz ${newsub}/ses-3T/anat/${newsub}_ses-3T_T1wT2wratio.nii.gz || true
+    git mv -fk ${subdir}/T1w/T2w_acpc_dc.nii.gz ${newsub}/ses-3T/anat/${newsub}_ses-3T_desc-dc_T2w.nii.gz || true
+    git mv -fk ${subdir}/T1w/T2w_acpc_dc_restore.nii.gz ${newsub}/ses-3T/anat/${newsub}_ses-3T_desc-dcrestore_T2w.nii.gz || true
+
+    # rename func files
+    #
+    # 3T rest1 LR
+    #
+    git mv -fk ${subdir}/MNINonLinear/Results/rfMRI_REST1_LR/Movement_AbsoluteRMS_mean.txt ${newsub}/ses-3T/func/${newsub}_ses-3T_task-rest_dir-LR_run-1_desc-MovementAbsoluteRMSmean.txt || true
+    git mv -fk ${subdir}/MNINonLinear/Results/rfMRI_REST1_LR/Movement_AbsoluteRMS.txt ${newsub}/ses-3T/func/${newsub}_ses-3T_task-rest_dir-LR_run-1_desc-MovementAbsoluteRMS.txt || true
+    git mv -fk ${subdir}/MNINonLinear/Results/rfMRI_REST1_LR/Movement_Regressors_dt.txt ${newsub}/ses-3T/func/${newsub}_ses-3T_task-rest_dir-LR_run-1_desc-MovementRegressorsdt.txt || true
+    git mv -fk ${subdir}/MNINonLinear/Results/rfMRI_REST1_LR/Movement_Regressors.txt ${newsub}/ses-3T/func/${newsub}_ses-3T_task-rest_dir-LR_run-1_desc-MovementRegressors.txt || true
+    git mv -fk ${subdir}/MNINonLinear/Results/rfMRI_REST1_LR/Movement_RelativeRMS_mean.txt ${newsub}/ses-3T/func/${newsub}_ses-3T_task-rest_dir-LR_run-1_desc-MovementRelativeRMSmean.txt || true
+    git mv -fk ${subdir}/MNINonLinear/Results/rfMRI_REST1_LR/Movement_RelativeRMS.txt ${newsub}/ses-3T/func/${newsub}_ses-3T_task-rest_dir-LR_run-1_desc-MovementRelativeRMS.txt || true
+    git mv -fk ${subdir}/MNINonLinear/Results/rfMRI_REST1_LR/ReclassifyAsNoise.txt ${newsub}/ses-3T/func/${newsub}_ses-3T_task-rest_dir-LR_run-1_desc-ReclassifyAsNoise.txt || true
+    git mv -fk ${subdir}/MNINonLinear/Results/rfMRI_REST1_LR/ReclassifyAsSignal.txt ${newsub}/ses-3T/func/${newsub}_ses-3T_task-rest_dir_LR_run-1_desc-ReclassifyAsSignal.txt || true
```

Comment:
might be worth checking how current version `fmriprep` stores any such related derivatives, and follow along.